plugin/trino-kafka/src/test/resources/protobuf-sources/unsupported_nested.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
```

Comment:
Can we have them in `protobuf` directory so that all the protobuf files would be in one directory ?

Reply:
Unfortunately, then generator will create objects for all proto files, but existing protos are intended to be used with DynamicMessage and these objects will be unneded/unused. On attempt to move existing protos to new folder unreachable for generator it will be need to refactor many reference places in few libs since those proto file location depends ProtobufUtils class package (since it's getting them by `getResource`, [see for details](https://github.com/trinodb/trino/blob/55f306bfc6d126a76c949cc29c2c7c2cca673ed7/lib/trino-record-decoder/src/main/java/io/trino/decoder/protobuf/ProtobufUtils.java#L128)). So that's looks like out of scope of the current issue.

Reply:
This will be solved if we parse the `proto` files and generate the `DynamicMessage` then we could have all the `proto` files in one directory.

Reply:
but we cannot due to https://github.com/trinodb/trino/pull/15312/commits/0710dce9df1fbc20259000ff8cd6edbfe4fbf92c#r1043458738